tests/rptest/remote_scripts/w_workload.py

```diff
@@ -0,0 +1,200 @@
+from asyncio import threads
+import sys
+
+from confluent_kafka import Producer, Consumer, KafkaException, KafkaError
+from confluent_kafka import TopicPartition, OFFSET_BEGINNING
+from flask import Flask, request
+from threading import Thread
+import logging
+import time
+
+
+class SyncProducer:
```

Comment:
This PR adds two new traffic generators, one in Java and one in Python -- what's the motivation for the different language choice?

Reply:
I've dropped the python